Assets/LeapMotion/Scripts/Utils/ValueMappingExtensions.cs

```diff
@@ -0,0 +1,125 @@
+﻿using UnityEngine;
+
+namespace Leap.Unity {
+
+  public static class ValueMappingExtensions {
```

Comment:
At least in my workflow, if it's the first time I'm using it in a file, I just type myFloat.Map, Ctrl+., Enter (auto-adds "using ValueMappingExtensions;"), then continue typing the function arguments. Or if you're a manual "using" typer, autocomplete will save the day as you write "using ValueM".